### PR TITLE
Fix import json_available error

### PR DESCRIPTION
**Pull request by crazyguitar**
Hi,

Flask 1.1.0 remove json_available for compatibility. Please refer to the Flask commit.

```diff
commit d362399e7ebfe167a27daef20e198ecabed1b49b
Author: Gabriel Saldanha <gabrielcrsaldanha@gmail.com>
Date:   Wed Jun 27 08:23:44 2018 -0300

    :fire: Remove code supposed to be removed at v1.0

diff --git a/flask/__init__.py b/flask/__init__.py
index 1ab13c36..2e5670f5 100644
--- a/flask/__init__.py
+++ b/flask/__init__.py
@@ -43,7 +43,3 @@ from . import json
 # This was the only thing that Flask used to export at one point and it had
 # a more generic name.
 jsonify = json.jsonify
-
-# backwards compat, goes away in 1.0
-from .sessions import SecureCookieSession as Session
-json_available = True
```

thanks,
chang-ning

### CODE REVIEW

**Comment by crazyguitar** [comment]
By the way, Is it necessary to support Python 3.3? I found Flask's compatibility with Python 3.3 is bad.

**Comment by lubomir-brindza** [comment]
Flask 1.1.0 dropped support for Python 3.4, so I suppose supporting it in a Flask extension doesn't make much sense.

**Comment by davidism** [comment]
This needs to be merged and released to support Flask 2.0.

**Comment by crazyguitar** [comment]
Hi,

**Comment by jnozsc** [comment]
cc @jcomo @jarus

File: flask_testing/utils.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 from werkzeug.utils import cached_property
 
 # Use Flask's preferred JSON module so that our runtime behavior matches.
```
Comment (jcomo) [inline flask_testing/utils.py:37]: Can you move this comment down to line 51 (where the import happens)?

**Comment by Garrett-R** [comment]
@jcomo seems like the there hasn't been a new version since 0.8.0 in March?  Would be great to have a new version that included this fix.

Great repo BTW, thanks!

**Comment by jcomo** [comment]
@Garrett-R thanks for the ping. I just uploaded 0.8.1 to pypi. Cheers!

**Comment by Garrett-R** [comment]
Awesome, thanks!

**Pull request merged**